packages/prosess-beregningsgrunnlag/src/components/ytelsesspesifikkseOpplysninger/frisinn/Inntektsopplysninger.jsx

```diff
@@ -22,21 +22,28 @@ const finnSamletBruttoForStatus = (andeler, status) => {
   return inntekt;
 };
 
+const finnVisningForStatus = (bg, status) => {
```

Comment:
Burde dette ha vore felles funksjonalitet sidan den samme koden ligg to stader?

Reply:
tenkte først ikke det, siden det før har vært forskjellige ting som regnes ut her og de nå tilfeldigvis har blitt lite. Men kan lage en utilklasse for det foreløpig